mod/likes/views/default/likes/count.php

```diff
@@ -19,19 +23,11 @@
 	$params = array(
 		'text' => $likes_string,
 		'title' => elgg_echo('likes:see'),
-		'rel' => 'popup',
-		'href' => "#likes-$guid",
-		'class' => 'elgg-non-link',
+		'class' => 'elgg-lightbox',
+		'href' => "ajax/view/likes/popup?guid=$guid",
```

Comment:
I'd prefer to have a data-view attribute. If JS breaks for some reason (or user clicks this early), the user will be sent to an ugly partial page.